device/core/src/device_client.ts

```diff
@@ -39,10 +42,11 @@ export class Client extends InternalClient {
    * @param {string}  connStr           A connection string (optional: when not provided, updateSharedAccessSignature must be called to set the SharedAccessSignature token directly).
    * @param {Object}  blobUploadClient  An object that is capable of uploading a stream to a blob.
    */
```

Comment:
>*/ [](start = 3, length = 2)


You appear to be missing the description of fileUploadApi in the parameter description.